objgen.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+from collections import defaultdict
+from ipaddress import IPv4Network, IPv6Network
+from typing import List
+
+
+class AnnouncementGenerator:
+    prefix: IPv4Network or IPv6Network
```

Comment:
I know this is OK for `dataclasses`, but not for normal classes. Just double check this is Pythonic.